src/presets/azure.ts

```diff
@@ -26,7 +26,18 @@ async function writeRoutes (nitro) {
     version: '2.0'
   }
 
+  let nodeVersion = '16'
+  try {
+    const currentNodeVersion = fse.readJSONSync(join(nitro.options.rootDir, 'package.json')).engines.node
```

Comment:
Isn't engines.node optional? Are we doing `try/catch` to ignore this?

Reply:
Implementation from https://github.com/unjs/nitropack/blob/305c49853d514ceb00342107a7affdbd9500f5f2/src/presets/firebase.ts#L57-L65.

But maybe might be better to just grab from `process`?

Reply:
I like the idea of falling back to process.versions :)